integration/spark2/src/main/spark2.2/org/apache/spark/sql/hive/CarbonSessionState.scala

```diff
@@ -325,7 +325,8 @@ class CarbonSqlAstBuilder(conf: SQLConf, parser: CarbonSpark2SqlParser, sparkSes
     val fileStorage = helper.getFileStorage(ctx.createFileFormat)
 
     if (fileStorage.equalsIgnoreCase("'carbondata'") ||
-        fileStorage.equalsIgnoreCase("'org.apache.carbondata.format'")) {
+      fileStorage.equalsIgnoreCase("carbondata") ||
```

Comment:
repeated logic

Reply:
It's different: 'carbondata'  and  carbondata.